### ISSUE TITLE
check-licenses test fails on release archives

### ISSUE THREAD

**Issue by ainola** [wmww/gtk-layer-shell#73]
Running `ninja test` on the latest release tarball presents the following error:

```
Traceback (most recent call last):
  File "/build/gtk-layer-shell/src/gtk-layer-shell-0.4.0/test/check-licenses.py", line 156, in <module>
    main()
  File "/build/gtk-layer-shell/src/gtk-layer-shell-0.4.0/test/check-licenses.py", line 121, in main
    all_files = get_important_files()
  File "/build/gtk-layer-shell/src/gtk-layer-shell-0.4.0/test/check-licenses.py", line 99, in get_important_files
    return get_files(get_project_root())
  File "/build/gtk-layer-shell/src/gtk-layer-shell-0.4.0/test/check-licenses.py", line 82, in get_files
    if ignored_by_git(search_path):
  File "/build/gtk-layer-shell/src/gtk-layer-shell-0.4.0/test/check-licenses.py", line 72, in ignored_by_git
    result = subprocess.run(
  File "/usr/lib/python3.8/subprocess.py", line 489, in run
    with Popen(*popenargs, **kwargs) as process:
  File "/usr/lib/python3.8/subprocess.py", line 854, in __init__
    self._execute_child(args, executable, preexec_fn, close_fds,
  File "/usr/lib/python3.8/subprocess.py", line 1702, in _execute_child
    raise child_exception_type(errno_num, err_msg, err_filename)
FileNotFoundError: [Errno 2] No such file or directory: 'git'
```

https://github.com/wmww/gtk-layer-shell/blob/4419b65ed8e074ceeff86e61d1c37ebdf6a0f002/test/check-licenses.py#L71

It'd be nice if tests can run without needing to use the git repository.

Thanks!

**Comment by wmww** [comment]
> It'd be nice if tests can run without needing to use the git repository.

Indeed. I don't think this is important enough to prompt another release now, but I'll fix this by next release. For now you can probably just patch out the license test.

**Comment by wmww** [comment]
I decided to only scan some toplevel directories to make life easier for packagers. Fixed better in #87.

**Comment by ainola** [comment]
Thank you very much, sir.
